TestApp/__mocks__/appcenter-crashes.js

```diff
@@ -0,0 +1,9 @@
+const Crashes = jest.mock('appcenter-crashes');
```

Comment:
Please catch up with develop and remove those. Might need to revisit the PR entirely now that #303 is merged.